unit-tests/live/safety/test-safety-metadata.py

```diff
@@ -55,7 +92,37 @@ def check_counter_and_timestamp_increase(frame, fps):
 pipe.stop()
 test.finish()
 
-################# Checking frame counter and timestamp increasing ##################
+################# Checking occupancy metadata required fileds are received ##################
+test.start("Checking Occupancy Stream Metadata received")
```

Comment:
Why some are capitalize (and some are not)?
I suggest `Checking occupancy stream metadata received`

Reply:
ok